docs/user-flows.md

```diff
@@ -75,8 +70,6 @@ Instead of providing a URL to navigate to, you can provide a callback function o
 
 This callback function _must_ perform an action that will trigger a navigation. Any interactions completed before the callback promise resolves will be captured by the navigation.
 
-The `startNavigation`/`endNavigation` functions _must_ surround an action that triggers a navigation. Any interactions completed after `startNavigation` is invoked and before `endNavigation` is invoked will be captured by the navigation.
```

Comment:
This is good, but can you remove the reference to `startNavigation` / `endNavigation` on L72